Ginger/GingerCoreCommon/Repository/BusinessFlowLib/ActivitiesGroup.cs

```diff
@@ -87,6 +87,11 @@ public string Description
             }
```

Comment:
do the ovveride to the ToString() on the RepositoryITemBase.cs and return "ItemName"